jwst/tests_nightly/general/miri/test_miri_steps_single.py

```diff
@@ -283,3 +282,97 @@ def test_miri_setpointing(self):
+        """
+        Regression test of masterbackgound subtraction with lrs, with user provided 1-D background
+        """
+
+        # input file has the background added
+        # Copy original version of file to test file, which will get overwritten by test
+        input_file = self.get_data(*self.test_dir,
+                                   'miri_lrs_sci+bkg_cal.fits',
+                                   docopy=True  # always produce local copy
+                                   )
+
+        # user provided 1-D background
+        input_1d_bkg_file = self.get_data(*self.test_dir,
+                                         'miri_lrs_bkg_x1d.fits',
+                                          docopy=True)
+
+        result = MasterBackgroundStep.call(input_file,
+                                           user_background=input_1d_bkg_file,
+                                           save_results=True)
+        # _____________________________________________________________________
+        # Test 1
+        # Run extract1D on the master background subtracted data (result)  and
+        # the science data with not background added
+
+        # run 1-D extract on results from MasterBackground step
+        result_1d = Extract1dStep.call(result, save_results=True)
+
+        # run 1-D extract on original science data without background
+        input_sci_cal_file = self.get_data(*self.test_dir,
+                                            'miri_lrs_sci_cal.fits')
+        # find the 1D extraction of this file
+        # find Extract1dStep on sci data to use the same version of
+        # this rountine run on both files  rather than running it off line
+        # and having the 1-D extracted science file stored as input to step
+
+        sci_cal_1d = Extract1dStep.call(input_sci_cal_file, save_results=True)
+
+        # Compare the MultiSpec 1-D data
+        atol = 500.  # set high for testing
+        rtol = 0.5  # set hight for testing
+        result_1d_data = result_1d.spec[0].spec_table['flux']
+        sci_cal_1d_data = sci_cal_1d.spec[0].spec_table['flux']
+
+        assert_allclose(result_1d_data, sci_cal_1d_data, atol=atol, rtol=rtol)
+        # _____________________________________________________________________
+        # Test 2
+        # Compare result (background subtracted image) to science image with no
+        # background. Subtract these images, smooth the subtracted image and
+        # the mean should be close to zero.
+
+        input_sci = datamodels.open(input_sci_cal_file)
+
+        # find the LRS region
+        bb = result.meta.wcs.bounding_box
+        x, y = grid_from_bounding_box(bb)
+        result_lrs_region = result.data[y.astype(int), x.astype(int)]
+        sci_lrs_region = input_sci.data[y.astype(int), x.astype(int)]
+
+        # do a 5 sigma clip on the science image
+        sci_mean = np.nanmean(sci_lrs_region)
+        sci_std = np.nanmean(sci_lrs_region)
+        upper = sci_mean + sci_std*3.0
```

Comment:
The comment (three lines up) says "5 sigma clip".
Also, do you want to clip on the low side as well?

Reply:
changed to 5 in code